src/server/datasource/ncbi.js

```diff
@@ -61,10 +63,17 @@ const processEntry = entryLine => {
     safeSplit( nodes[ NODE_INDICES.OTHER_DESIGNATORS ] )
   );
 
+  let dbXrefs = safeSplit( nodes[ NODE_INDICES.DB_XREFS ] ).map( xref => {
+    let [ db, id ] = xref.split( /:(.+)/ );
```

Comment:
This line is confusing to me at first glance.  Is it sufficient to just `split(':')`?

Reply:
The format is `db:id | db:id | ... ` but some databases have colons in their actual IDs, like HGNC.

For example: `MIM:605312|HGNC:HGNC:30142|Ensembl:ENSG00000130513`

Reply:
OK, I thought there may be some edge cases here.  It may be worth adding an explanation in the comment on that line.